### DISCUSSION TITLE
Is it possible to set it stop opening mutiple browser windows

### DISCUSSION THREAD

**Discussion by PhantomICEMAN** [jef/streetmerchant#2046] [Ideas and feature requests]
Is there a way to set it it up so that it doesnt keep opening mutiple  browser windows if it adds an item to the cart as it then become counter productive

**Comment by PMVX** [comment]
Update your dotenv to show: OPEN_BROWSER=false

**Reply by ordinafdim** [reply]
If we do that will it open a page at all?

**Reply by PMVX** [reply]
No. It wont open the browser even if it finds something in stock.
the HEADLESS variable by default is set to true which hides the browser.
If you set HEADLESS=false you will see the browser open up with lots of tabs.
